ivy/functional/ivy/nest.py

```diff
@@ -1055,6 +1055,61 @@ def nested_map(
     ret
         x following the applicable of fn to it's nested leaves, or x itself if x is not
         nested.
+
+    Examples
+    --------
+    With :class:`Tuple` inputs:
+
+    >>> x = ([[1., 2.], [3., 4.]])
+    >>> function = lambda a : a * 2
+    >>> ivy.nested_map(x, function)
+    >>> print(x)
+    ([[2.0, 4.0], [6.0, 8.0]])
```

Comment:
No need to use `print(x)`

```suggestion
    >>> ivy.nested_map(x, function)
    ([[2.0, 4.0], [6.0, 8.0]])
```

Although I know you might want to show that `x` have actually been changed, so this can also be written (tbh I prefer it more)
```suggestion
    >>> ivy.nested_map(x, function)
    ([[2.0, 4.0], [6.0, 8.0]])
    >>> print(x)
    ([[2.0, 4.0], [6.0, 8.0]])
```